google/cloud/aiplatform/explain/lit.py

```diff
@@ -0,0 +1,256 @@
+        'Please install the SDK using "pip install pip install python-aiplatform[lit]"'
+    )
+
+if "pandas" not in sys.modules:
+    raise ImportError(
+        "Pandas is not installed and is required to read the dataset. "
+        'Please install Pandas using "pip install python-aiplatform[lit]"'
+    )
+
+
+class _VertexLitDataset(lit_dataset.Dataset):
+    """LIT dataset class for the Vertex LIT integration.
+
+    This is used in the create_lit_dataset function.
+    """
+
+    def __init__(
+        self,
+        dataset: "pd.Dataframe",  # noqa: F821
+        column_types: OrderedDict[str, "lit_types.LitType"],  # noqa: F821
+    ):
+        """Construct a VertexLitDataset.
+        Args:
+          dataset:
+            Required. A Pandas Dataframe that includes feature column names and data.
+          column_types:
+            Required. An OrderedDict of string names matching the columns of the dataset
+            as the key, and the associated LitType of the column.
+        """
+        self._examples = dataset.to_dict(orient="records")
+        self._column_types = column_types
+
+    def spec(self):
+        return dict(self._column_types)
+
+
+class _VertexLitModel(lit_model.Model):
+    """LIT model class for the Vertex LIT integration.
+
+    This is used in the create_lit_model function.
+    """
+
+    def __init__(
+        self,
+        model: str,
+        input_types: OrderedDict[str, "lit_types.LitType"],  # noqa: F821
+        output_types: OrderedDict[str, "lit_types.LitType"],  # noqa: F821
+    ):
+        """Construct a VertexLitModel.
+            Args:
+              model:
+                Required. A string reference to a local TensorFlow saved model directory.
+                The model must have at most one input and one output tensor.
+              input_types:
+                Required. An OrderedDict of string names matching the features of the model
+                as the key, and the associated LitType of the feature.
+              output_types:
+                Required. An OrderedDict of string names matching the labels of the model
+                as the key, and the associated LitType of the label.
+        """
+        self._loaded_model = tf.saved_model.load(model)
+        serving_default = self._loaded_model.signatures[
+            tf.saved_model.DEFAULT_SERVING_SIGNATURE_DEF_KEY
+        ]
+        _, self._kwargs_signature = serving_default.structured_input_signature
+        self._output_signature = serving_default.structured_outputs
+
+        if len(self._kwargs_signature) != 1:
+            raise ValueError("Please use a model with only one input tensor.")
+
+        if len(self._output_signature) != 1:
+            raise ValueError("Please use a model with only one output tensor.")
+
+        self._input_types = input_types
+        self._output_types = output_types
+
+    def predict_minibatch(
+        self, inputs: List["lit_types.JsonDict"]
```

Comment:
Since the modules are now imported we can remove the forward references for those modules throughout:

```suggestion
        self, inputs: List[lit_types.JsonDict]
```

Reply:
Changed all instances to remove forward references.